wiki/Tournaments/UKCC/11/en.md

```diff
@@ -0,0 +1,121 @@
+- Seeding will be done based on the sum of normalised scores from qualifier maps.
+- All staff (apart from the hosts, mappool selectors and referees) are allowed to participate in the tournament.
+- All players are to treat staff and other participants with a high level of respect. Any kind of mistreatment on the Discord server or during matches (this includes homophobia, racism and toxicity) will result in a **ban** and the offending player will be **blacklisted** from participating from future UKCC tournaments.
+- **Where rules do not prevail, common sense shall. The administration has explicit discretion to apply their judgement on this as they see fit.**
+- Players are **required** to join the public Discord server so that they are contactable by other players and staff. If a player can't be contacted or is not responding to messages through Discord, in-game messages, or forum PMs, contact the staff.
+- Mappools are to be released the Monday before a stage is due to happen.
+
+### Reschedules
+
+- Schedules for the next round are released on the Monday before a stage is due to happen and **all** matches will be scheduled for the weekend. You can reschedule for weekdays at the staff's discretion.
+- In order to reschedule, you must contact your opponents, agree on a time, and be able to provide **proof** (by form of a screenshot/image) of the agreed reschedule to the hosts.
+- In effort to ensure that as many matches take place as possible, reschedules are allowed up to **2 hours before your match is due to be played**. However, your opponent holds the right to **refuse a reschedule if asked less than 12 hours from the scheduled time**.
+- If it is impossible to find a time to reschedule to that suits both teams, both teams should talk to the hosts (![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973), ![][flag_GB] [Damarsh](https://osu.ppy.sh/users/7465147) or ![][flag_TR] [oralekin](https://osu.ppy.sh/users/7631823)) who will attempt to sort something out.
+- Matches that are uncertain to happen (for example, losers' bracket matches) will still appear on the schedule. They are to be rescheduled if needed in the same way as confirmed matches.
+- If a match needs rescheduling to a time after a related losers' bracket match, the matches relying on the outcome of the rescheduled match will be rescheduled by staff and 24 hours will be given for rescheduling them, if needed.
+
+### Pre-match
+
+- Referees will create the lobby approximately **10 minutes** before the match is due to start.
+- Teams have a 10 minute grace period after their scheduled match time to join the lobby. If a player fails to join the lobby in this time, then the default win will go to the other player. If both teams do not show up, both teams will recieve losses.
+- We take no-shows seriously, if you fail to show up to your match without valid reason, you may forfeit your right to future UKCC tournaments.
+- The referee cannot play during the match.
```

Comment:
unless you're actually somehow emphasising the fact that a referee shouldn't play solo or something while refereeing a match (is that even possible?)

```suggestion
- The referee cannot play in the match.
```

Reply:
To a certain degree, both, a referee shouldn't be playing directly in the match but it wouldn't be a good idea for them to also be playing their own stuff while the match is in progress just in case. I'd say the suggestion is closer to the literal meaning though so I'll use that.